src/components/Player/Track.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { InternationalString } from "@hyperion-framework/types";
+import { LabeledResource } from "hooks/use-hyperion-framework/getSupplementingResources";
+import { getLabel } from "hooks/use-hyperion-framework";
+
+export interface TrackProps {
+  resource: LabeledResource;
+  ignoreCaptionLabels: string[];
```

Comment:
Could we make `ignoreCaptionLabels` optional?  And then when we destructure the props, set a default value below like:

```
const Track: React.FC<TrackProps> = ({ resource, ignoreCaptionLabels = [] }) => {
```

Reply:
Yeah, I did this so that the type def was always an array.